<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterFactoryParametersDialog</class>
 <widget class="QDialog" name="EnterFactoryParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Factory Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="factoryParametersInstructions_label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Please enter the assembly date and time, and serial number of this gimbal, then press OK to commit these values to flash&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="assemblyDate_label">
     <property name="text">
      <string>Assembly Date (MM/DD/YYYY):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="assemblyDate">
     <property name="inputMask">
      <string>09/09/9999;_</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="assemblyTime">
     <property name="inputMask">
      <string>09:09:09;_</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="assemblyTime_label">
     <property name="text">
      <string>Assembly Time: (HH/MM/SS)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLabel" name="settingFactoryParameters_label">
     <property name="text">
      <string>Setting Parameters...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QGroupBox" name="serialNumberGroup">
     <property name="title">
      <string>Serial Number</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="serialNumber_label">
        <property name="text">
         <string>Serial Number (5 Digits):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="languageCountry"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="languageCountry_label">
        <property name="text">
         <string>Language/Country:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="generatedSerialNumber_label">
        <property name="text">
         <string>Generated Serial Number:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="generatedSerialNumber">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="serialNumber">
        <property name="inputMask">
         <string>99999;_</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>assemblyDate</tabstop>
  <tabstop>assemblyTime</tabstop>
  <tabstop>languageCountry</tabstop>
  <tabstop>serialNumber</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EnterFactoryParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EnterFactoryParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
